
# order is FRONT_LEFT, BACK_LEFT, FRONT_RIGHT, BACK_RIGHT
# speed  [m/sec]
float32[4] speed

# pos   [m]
float32[4] position

# last pid time, TODO TOGLIERE
float32 lastPidTime

